pkg/uuid/uuid.go

```diff
@@ -16,7 +16,13 @@ Copyright 2023 freiheit.com*/
 
 package uuid
 
-import "4d63.com/uuid/uuidv1"
+import (
+	"github.com/onokonem/sillyQueueServer/timeuuid"
```

Comment:
Is there a reason why you're not using the more standard github.com/google/uuid? Seems like it also supports UUIDv1.

Reply:
I got the recommendation to not use it.
I just now tried it out briefly, and it does not seem as easy to use, e.g. it brings it's own "time" type.